<?xml version="1.0" encoding="UTF-8"?>
<problems>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_travis.py</file>
    <line>3</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_travis.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'itertools'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_travis.py</file>
    <line>3</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_travis.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'combinations'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_travis.py</file>
    <line>61</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_travis.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'xrange'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_travis.py</file>
    <line>87</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_travis.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'len'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>3</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>No module named sys</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>4</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'itertools'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>4</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'combinations'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>5</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'os'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>5</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'uname'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>6</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'multiprocessing'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>6</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'cpu_count'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>18</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'xrange'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>18</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'len'</description>
  </problem>
  <problem>
    <file>file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py</file>
    <line>63</line>
    <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/deps/jemalloc/scripts/gen_run_tests.py" />
    <problem_class severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
    <description>Unresolved reference 'len'</description>
  </problem>
</problems>

